### DISCUSSION TITLE
Conexão com o Banco de Dados

### DISCUSSION THREAD

**Discussion by Igao02** [balta-io/forum#441] [Tira Dúvidas] [answered]
Boa noite Balta,
Eu criei o projeto porém com o Blazor Auto. Eu consegui fazer a conexão com o banco de dados, porém as tabelas Product e Category não estão aparecendo, o que estou fazendo de errado?? 
![Banco](https://github.com/balta-io/forum/assets/88517354/a7cdfa13-6da9-4d4d-8a45-d35fd9f00fe8)

![Conexão](https://github.com/balta-io/forum/assets/88517354/1ba606b6-bbe9-42e0-b29b-781d1ef71f41)

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @Igao02, como vai?
Você executou as migrações do EF?

**Reply by Igao02** [reply]
Bom dia Balta,
Sim executei tudo que foi mostrado nas aulas anteriores. A migracao não dá erro, só não aparece as tabelas no meu banco.

**Reply by Igao02** [reply]
Balta, consegui.
Segue a string de conexão:
![image](https://github.com/balta-io/forum/assets/88517354/7176c542-13fc-4229-8e47-6fa4bcb5fb60)

**Reply by andrebaltieri** [reply]
Show!!! Meus parabéns 🚀
